charts/airflow/values.yaml

```diff
@@ -232,6 +232,10 @@ airflow:
   ##
   extraVolumes: []
 
+  ## kubernetes cluster domain name configured in the kubelet with clusterDomain option
```

Comment:
I think this docstring is more clear if it's worded like:
```
## kubernetes cluster domain name (configured in the kubelet with `--cluster-domain` flag)
```

Reply:
Agree but I found that this flag is deprecated and should be moved to configuration file as described here: https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/

Reply:
How about docstring from my second commit?